### PR TITLE
docs: inject chainable methods (#1917)

### PR DESCRIPTION
**Pull request by alemagio**
Added a reference to chainable methods on 'inject' function based on the update of 'light-my-request'

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Testing.md [hunk 1]
```diff
@@ -21,6 +21,19 @@ fastify.inject({
 })
 ```
 
+you can also use chaining methods if you do not pass the callback function
```
Comment (jsumners) [inline docs/Testing.md]:
```suggestion
`.inject` methods can also be chained by omitting the callback function:
```

File: docs/Testing.md [hunk 2]
```diff
@@ -21,6 +21,19 @@ fastify.inject({
...
+
+```js
+fastify
+  .inject()
+  .get('/')
+  .headers({ foo: 'bar' })
+  .query({ foo: 'bar' })
+  .end((err, res) => {
```
Comment (Eomm) [inline docs/Testing.md]:
```suggestion
  .end((err, res) => { // the .end call will trigger the request
```

**Comment by alemagio** [comment]
Sorry guys, so i add both the changes and we are good to go?

**Comment by jsumners** [comment]
@alemagio did you forget to push the changes?

**Comment by alemagio** [comment]
> @alemagio did you forget to push the changes?

Nono, i didn't get if you want me to apply both of the suggestions

**Comment by jsumners** [comment]
They are suggestions for improvement and one reviewer said the PR will be acceptable with them applied.

**Comment by alemagio** [comment]
Ok, great.
Pushing in minutes

**Pull request merged**